diff --git a/node_modules/react-native/Libraries/Components/RefreshControl/__mocks__/RefreshControlMock.js b/node_modules/react-native/Libraries/Components/RefreshControl/__mocks__/RefreshControlMock.js
new file mode 100644
index 0000000..2ad6867
--- /dev/null
+++ b/node_modules/react-native/Libraries/Components/RefreshControl/__mocks__/RefreshControlMock.js
@@ -0,0 +1,9 @@
+'use strict';
+
+const React = require('react');
+
+function RefreshControlMock(props) {
+  return React.createElement('RefreshControl', props, props.children);
+}
+
+module.exports = RefreshControlMock;
